### PR TITLE
Update index.rst to include MAD info

### PR DESCRIPTION
**Pull request by johnsolk**
Might need to update time/day if people object to Wednesdays (spring quarter 2018)

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -14,12 +14,12 @@ taken from the `Next-Gen Sequence Analysis Workshops
...
 <http://datacarpentry.org>`__.
 
-* All workshops are on UC Davis campus unless otherwise specified.
+* All workshops and community events are on UC Davis campus unless otherwise specified.
 * Workshops are open to everyone whether or not they are part
   of the UC Davis community, subject to space constraints.
 * Workshops are free unless otherwise specified.
 
-If you want to be notified of new workshops you can sign up for the
+If you want to be notified of new workshops or events happening in the data science community at UC Davis, you can sign up for the
```
Comment (karenword) [inline index.rst]: I'm a little concerned about confusion/overlap with the DSI community. Maybe just "events" here?

File: index.rst [hunk 2]
```diff
@@ -30,6 +30,14 @@ You can subscribe to our Google Calendar containing all of our events
 Or, you can simply view the calendar `here <https://calendar.google.com/calendar/embed?src=mqcgc13fqifh9ocb7l8novguik%40group.calendar.google.com&ctz=America/Los_Angeles>`__.
 
 
+Meet and Analyze Data (MAD)
+----------------------------
+Weekly meetup with the data science community at UC Davis in the :doc:`Bennett Conference room <Bennett-room>` on the Veterinary Medicine campus from 3-5 pm on Wednesdays (updated Spring quarter 2018).
+
+* Have questions? Come, bring your questions! There are no silly questions. Researching your own answers prior to showing up is not necessary.
+* Have skills? Come! This is a relaxed co-working space with a group of friendly, computationally-minded people who enjoy discussing a diversity of data analysis problems and methods.
+* Want to come work in silence and eavesdrop on other peoples' struggles to gain inspiration? Great!
```
Comment (karenword) [inline index.rst]: As much as I would like to say "eavesdrop" (and I probably suggested this) I think it is maybe more diplomatic to say "listen in on problem-solving sessions to gain inspiration"

**Comment by johnsolk** [comment]
Thanks @karenword, modified wording!

**Review by raynamharris — APPROVED** [review]

**Comment by ctb** [comment]
(@raynamharris @karenword @ljcohen question - do you not have merge privileges?)

**Comment by johnsolk** [comment]
I do. But thought it was against protocol to merge my own PR?

**Comment by raynamharris** [comment]
I have merge privileges. I thought I would first make an approval. Next time I'll approve and merge so that you don't have to merge your own PR.

**Comment by ctb** [comment]
as long as it's been looked at by a second brain, we're good to go!

**Comment by ctb** [comment]
once reviewed => merge!!

**Comment by johnsolk** [comment]
Cool, will merge next time after review. Thanks!

**Pull request merged**